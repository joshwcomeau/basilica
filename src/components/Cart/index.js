// eslint-disable-next-line no-unused-vars
import React, { Component, PropTypes } from 'react';
import classNames from 'classnames';

import './index.scss';


const Cart = () => {
  const classes = classNames(['cart']);

  return (
    <div className={classes}>
      <h2>
        Your Cart
        <span className="item-count">2 Items</span>
      </h2>
    </div>
  );
};

Cart.propTypes = {

};

Cart.defaultProps = {

};

export default Cart;
