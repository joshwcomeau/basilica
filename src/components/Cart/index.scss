@import '../../variables';

.cart {


  @media screen and (max-width: $mq-tablet) {

  }

  @media screen and (max-width: $mq-mobile) {

  }
}
