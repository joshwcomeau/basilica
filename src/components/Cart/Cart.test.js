/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import Cart from './index';

describe('Cart', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Cart />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
