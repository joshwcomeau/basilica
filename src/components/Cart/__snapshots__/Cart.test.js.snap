exports[`Cart renders correctly 1`] = `
<div
  className="cart">
  Your Component Here :)
</div>
`;
